node-fs-readfile/cat.js

```diff
@@ -0,0 +1,16 @@
+import { readFile } from 'node:fs/promises';
+
+const [, , ...argList] = process.argv;
+
+async function readAny(fileName) {
+  try {
+    const file = await readFile(fileName, 'utf-8');
```

Comment:
This isn't actually guaranteed to read and log all the files in order. One of the later ones in the list could finish reading before an earlier one. Try running it with a lengthy file (like hipster-ipsum.txt) listed before a short file (like hopper.txt). You'll see they will sometimes come out in the wrong order.